build.sh

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Could you please, change `build python-template template/scala` to `build scala-template template/scala` at [line 21](https://github.com/big-data-europe/docker-spark/blob/1c9faf52b82c2fbbb526ccdc3fc0cdc97ce412f2/build.sh#L21)?

Reply:
It looks like this line is commented out on master branch. Would you mind if I change it on this branch?

Reply:
Sure, feel free to change it here!